Dockerfile

```diff
@@ -2,10 +2,8 @@ FROM node:14-slim
 
 WORKDIR /app
 
-# Install dependencies, but don't generate lock files as we want to always fetch
-# the lastest @mdn/browser-compat-data package.
 COPY package.json ./
-RUN npm install --no-package-lock && \
+RUN npm install --no-dev && \
```

Comment:
I can't find `--no-dev` in the documentation of npm install, maybe we can just do `npm install`

Reply:
thanks, reverted changes :+1: